rush-plugins/rush-audit-cache-plugin/src/core/Analyzer.ts

```diff
@@ -84,26 +71,52 @@ export class AuditCacheAnalyzer {
         if (!project) {
           throw new Error(`no project ${projectName} RushConfigurationProject`);
         }
+        const readFileResolver: ReadFileResolver = new ReadFileResolver();
+        const writeFileResolver: WriteFileResolver = new WriteFileResolver();
+
         const { projectFolder } = project;
 
+        // get user project rush audit cache config
+        const rushProjectAuditCacheJsonPath: string = path.join(
+          projectFolder,
+          RUSH_AUDIT_CACHE_JSON_RELATIVE_PATH
+        );
+
+        const rushProjectAuditCacheJson: IAuditCacheFileFilter | undefined =
+          tryLoadJson<IAuditCacheFileFilter>(rushProjectAuditCacheJsonPath);
+
+        const { fileFilters = [] } = rushProjectAuditCacheJson ?? {
+          fileFilters: [],
+        };
```

Comment:
not important, maybe it's better to write `const fileFilters: IAuditCacheFileFilter['fileFilters'] = rushProjectAuditCacheJson?.filteFilters || []`